lib/index.js

```diff
@@ -26,8 +26,11 @@ import { queryParameters } from './utils'
  * ```
  */
 
+let _client
 const ZAFClient = {
   init: function (callback, loc) {
+    if (_client) return _client
```

Comment:
With this change, we would always return the same client regardless of different `callback` or `loc` being passed, which doesn't look right to me. I've seen apps creating multiple clients in the past during app review process, not entirely sure about their use case but I think this would be a risky change if we suddenly remove this capability.